README.md

```diff
@@ -120,17 +120,38 @@ To create a new release branch:
 
 In the `main` branch:
 
-1. Update the versions list in [params.toml](/config/_default/params.toml) to add the next release.
+1. Add a new menu version corresponding to the new release branch in [params.toml](/config/_default/params.toml) that points to https://kyverno.io below these lines:
+
+```toml
+# version_menu = "Versions"
+# Add your release versions here
+[[menu.versions]]
```

Comment:
I can't find this. Is it `version_menu = "v1.13.0"`?

Reply:
No, it's the section for menu entries. I copy pasted it. Should be the last thing in the file.